airflow/providers/google/cloud/operators/vertex_ai/custom_job.py

```diff
@@ -1275,8 +1324,77 @@ def execute(self, context: Context):
             self.log.info("The Custom Job ID %s does not exist.", self.custom_job)
 
 
+class CancelCustomTrainingJobOperator(GoogleCloudBaseOperator):
```

Comment:
This operator is not necessary we have on_kill method for each operator which runs cancel_job

Reply:
fixed